sabnzbd/newsunpack.py

```diff
@@ -2077,6 +2077,20 @@ def quick_check_set(set, nzo):
             if nzf.md5sum == md5pack[file]:
                 try:
                     logging.debug("Quick-check will rename %s to %s", nzf.filename, file)
+
+                    # Let's check if there is a subdir in 'file' (only needed on POSIX so '/')
+                    new_path = re.search("(.*)/", file)
```

Comment:
what if the post is created on Windows? Will it contain a "/' or a Windows style "\"? 
Change to os.path.sep does not solve it, AFAIK

Reply:
The move fails on Windows as well, but it uses "Parchive 2.0 client version 1.3.1.3 by Yutaka Sawada". It creates the directory and moves the the files. Yes, it's \ in Windows unless you use bash for Windows.